go/sharedutils/util.go

```diff
@@ -267,3 +267,22 @@ func CleanIP(s string) (string, error) {
 		return ip.String(), nil
 	}
 }
+
+func RemoveDuplicates(elements []string) []string {
+	// Use map to record duplicates as we find them.
+	encountered := map[string]bool{}
+	result := []string{}
+
+	for v := range elements {
+		if encountered[elements[v]] == true {
```

Comment:
Could just negate this and move the content of the else in here instead of having an empty if body